src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -153,7 +115,7 @@ public function __construct(
             $this->getStagingStorageInput(),
             $this->component,
             $this->configId,
-            isset($this->runtimeConfig['backend']) ? $this->runtimeConfig['backend'] : []
+            $this->runtimeConfig['backend'] ?? []
```

Comment:
On moze byt `null`? Akoze null je validna hodnota pre ten backend alebo preco si to takto upravil?

Reply:
To je to stejný ne?

Reply:
Sorry, ja som sa na to dival nejak obracene :) Ale myslel som, ze pripade isset() moze byt ta hodnota null, ale nemoze, je to stejny.